### PR TITLE
1.5 Correction in the definition of Big O notation

### PR DESCRIPTION
**Pull request by abhrac**
# Changes made
Corrected the relation specified between f(x) and g(x) in the definition of Big O notation.

# Justification
f(x) = O(g(x)) implies that f(x) is upper bounded by Mg(x), not the other way round.
https://en.wikipedia.org/wiki/Big_O_notation#Formal_definition

### CODE REVIEW

File: content/ch-states/case-for-quantum.ipynb [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "  <summary>Definition: Big O notation (Click to expand)</summary>\n",
     "For some example functions $f(x)$ and $g(x)$ and parameter $x$, the statement $f(x) = O(g(x))$  means that there exists some finite numbers $M>0$ and $x_0$ such that\n",
```
Comment (3yakuya) [inline content/ch-states/case-for-quantum.ipynb:60]: Could we update this to "exist" rather than exists, since we are referring to numbers?

**Review by 3yakuya — DISMISSED** [review]
One minor comment that would be good to improve here, important catch otherwise, could be very confusing for readers unfamiliar with complexity theory!

File: content/ch-states/case-for-quantum.ipynb (hunk 1 above)
Comment (abhrac) [reply content/ch-states/case-for-quantum.ipynb]: Addressed.

**Review by frankharkins — APPROVED** [review]
Looks good, thank you!

**Pull request merged**